### PR TITLE
Demo practice

### PR DESCRIPTION
**Pull request by ademcigdem**

### CODE REVIEW

**Review by srknml — APPROVED** [review]
👍

**Review by IrfanAzim34 — APPROVED** [review]

**Review by ustamehmet83 — COMMENTED** [review]
please why do you write this line

File: src/test/java/HelloWorld.java [hunk 1]
```diff
@@ -4,6 +4,8 @@ public static void main(String[] args) {
         System.out.println("Hello new World");
         System.out.println("Group-24 was here");
         System.out.println("Group-24 was here second time");
+        System.out.println("Group-24 was here always");
+        System.out.println("Group-24 was here sometimes");
         System.out.println("Zubeyir was here second time");
```
Comment (zubeyrdeveli) [inline src/test/java/HelloWorld.java:9]: you can delete this line

**Review by zubeyrdeveli — COMMENTED** [review]
thanks

**Review by idrisdogruoz — DISMISSED** [review]
??

File: src/test/java/HelloWorld.java (hunk 1 above)
Comment (ademcigdem) [reply src/test/java/HelloWorld.java:9]: ok sure

**Review by MuratYasul — APPROVED** [review]

**Pull request merged**